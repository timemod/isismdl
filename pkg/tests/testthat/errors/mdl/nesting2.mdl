ident EQ1 X1 = Sum(J = 3, 4 : J * Sum(J = -2, -1 : J * Y ) );
