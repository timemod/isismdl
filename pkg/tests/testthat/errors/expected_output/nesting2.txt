Nested sum not allowed

Line: 1 of file mdl/nesting2.mdl
ident EQ1 X1 = sum(J = 3, 4 : J * sum(J = -2, -1 : J * Y ) );
                                                ^

